OpenTissue/gpu/image/io/image_write.h

```diff
@@ -1,64 +1,117 @@
-#ifndef OPENTISSUE_GPU_IMAGE_IO_IMAGE_WRITE_H
-#define OPENTISSUE_GPU_IMAGE_IO_IMAGE_WRITE_H
 //
 // OpenTissue Template Library
 // - A generic toolbox for physics-based modeling and simulation.
 // Copyright (C) 2008 Department of Computer Science, University of Copenhagen.
 //
 // OTTL is licensed under zlib: http://opensource.org/licenses/zlib-license.php
 //
-#include <OpenTissue/configuration.h>
-
-#include <OpenTissue/gpu/image/image.h>
-#include <OpenTissue/gpu/image/io/image_il_wrap.h>
+#pragma once
```

Comment:
Is this standard? I am concerned about cross-compiler support on multiple plaltforms

Reply:
It is actually non-standard but well supported by most compilers. While more and more libraries are starting to use this I understand your concern.